### DISCUSSION TITLE
How to reboot coolify after a server downtime/crash?

### DISCUSSION THREAD

**Discussion by mbeaudru** [coollabsio/coolify#2074] [Q&A] [answered]
Hi 👋 

In a case where coolify is self-hosted, I didn't find in the docs how to restart coolify to the way it was before a server crash? Is it automatic or do we need to run a specific command?

Thanks!

**Comment by mbeaudru** [comment, marked as answer]
For anyone tumbling here wondering about the same thing, turns out docker instances are already configured to auto-restart when the server reboots.

List server instance: `docker ps -a`

Find the coolify container id and then:

`docker inspect -f '{{ .HostConfig.RestartPolicy.Name }}' <container_id>`

Should prompt `always` !

----

I just did a full server reboot and when it started back everything was started automatically again and all services were up 👍 

@andrasbacsai just to check twice 🙏  will close if you confirm!

**Reply by andrasbacsai** [reply]
Yes, if the Docker Engine is started, it will bring up all the containers (Coolify or deployed resources via Coolify) automatically.

If in some cases Coolify is not started automatically, you can execute the installation script. It will detect that you already have it installed, so it will just bring it up (and update if there is a new version).
